app/models/work_version.rb

```diff
@@ -293,6 +293,15 @@ def all_dois
     super
   end
 
+  def thumbnail_url
+    auto_generate_thumbnail? ? file_resources&.map { |fr| fr&.thumbnail_url }&.last : nil
+  end
```

Comment:
Did this get added in by mistake? I'm not sure why this is part of the PR.

Reply:
yes, probably missed it when rebasing, I will clean it up. thanks for catching it.

Reply:
ok should be cleaned up now.